code/modules/security levels/security levels.dm

```diff
@@ -1,61 +1,61 @@
 /var/security_level = 0
-//0 = code green
-//1 = code blue
-//2 = code red
-//3 = code delta
+//0 = daybreak protocol
+//1 = sunset protocol
+//2 = nightfall protocol
+//3 = supernova protocol
 
 //config.alert_desc_blue_downto
 
 /proc/set_security_level(var/level)
 	switch(level)
-		if("green")
-			level = SEC_LEVEL_GREEN
-		if("blue")
-			level = SEC_LEVEL_BLUE
-		if("red")
-			level = SEC_LEVEL_RED
-		if("delta")
-			level = SEC_LEVEL_DELTA
+		if("daybreak")
+			level = SEC_LEVEL_DAYBREAK
+		if("sunset")
+			level = SEC_LEVEL_SUNSET
+		if("nightfall")
+			level = SEC_LEVEL_NIGHTFALL
+		if("supernova")
+			level = SEC_LEVEL_SUPERNOVA
 
 	//Will not be announced if you try to set to the same level as it already is
-	if(level >= SEC_LEVEL_GREEN && level <= SEC_LEVEL_DELTA && level != security_level)
+	if(level >= SEC_LEVEL_DAYBREAK && level <= SEC_LEVEL_SUPERNOVA && level != security_level)
 		switch(level)
-			if(SEC_LEVEL_GREEN)
+			if(SEC_LEVEL_DAYBREAK)
 				world << sound('sound/misc/notice2.ogg')
-				to_chat(world, "<font size=4 color='red'>Attention! Security level lowered to green</font>")
+				to_chat(world, "<font size=4 color='red'>Attention! Sunset Protocol has been lowered to Daybreak.</font>")
```

Comment:
This also feels a bit clunky. What's the direct replacement for the general concept of "security level" in this system? Alternatively, I suggest rephrasing the whole message.